csharp/src/SeedLang.Shell/Program.cs

```diff
@@ -13,20 +13,83 @@
 // limitations under the License.
 
 using System;
+using CommandLine;
+using CommandLine.Text;
 using SeedLang.Common;
 
 namespace SeedLang.Shell {
   class Program {
-    static void Main(string[] _) {
+    class Options {
+      [Option('r', "rule", Required = false, Default = "statement",
+        HelpText = "The parse rule of the source code." +
+                   " Allowed values are identifier, number, string, expression or statement.")]

```

Comment:
Optional: consider to auto-generate this allowed value list from enum.GetName().